src/common/log.h

```diff
@@ -75,12 +75,16 @@ inline std::wstring get_stack_trace()
     return L"";
 }
 
-#define CASPAR_LOG_CURRENT_EXCEPTION()                                                                                 \
+#define CASPAR_LOG_CURRENT_EXCEPTION_LEVEL(lvl)                                                                        \
     try {                                                                                                              \
-        CASPAR_LOG(error) << L"Exception: " << caspar::u16(::caspar::log::current_exception_diagnostic_information()) << L"\r\n" << caspar::log::get_stack_trace();                   \
+        CASPAR_LOG(lvl) << L"Exception: " << caspar::u16(::caspar::log::current_exception_diagnostic_information())    \
+                        << L"\r\n";                                                                                    \
+        CASPAR_LOG(lvl) << caspar::log::get_stack_trace();                                                             \
```

Comment:
Why split this into two?

Reply:
Oops. that was from when I was trying something. Ill recombine it